src/mumble/MainWindow.h

```diff
@@ -269,6 +269,9 @@ class MainWindow : public QMainWindow, public MessageHandler, public Ui::MainWin
 		void pttReleased();
 		void whisperReleased(QVariant scdata);
 		void onResetAudio();
+		/// Called whenever the self-mute state may have changed. Dispatches
+		/// corkAudioInputStream() if the desire for audio input has changed
```

Comment:
I would add a period at the end of the sentence, for consistency with the rest of the documentation.